packages/compare-slider-block/src/components/EditorOverlay/EditorOverlay.tsx

```diff
@@ -0,0 +1,101 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconArrowCircleUp20, IconImageStack20, IconTrashBin16 } from '@frontify/fondue';
+import { BlockItemWrapper } from '@frontify/guideline-blocks-shared';
+import { Alignment, EditorOverlayProps, SliderImageSlot } from '../../types';
+
+export const EditorOverlay = ({
+    alignment,
+    openAssetChooser,
+    startFileDialogUpload,
+    firstAsset,
+    secondAsset,
+    borderStyle,
+    handleAssetDelete,
+    renderLabel,
+}: EditorOverlayProps) => {
+    return (
+        <>
```

Comment:
Fragment not needed

Reply:
It is actually needed. There are 3 children here, if I add a wrapper div here the layout crashes